lib/mongoHandler.js

```diff
@@ -162,6 +178,8 @@ MongoStore.prototype.initialise = function(resourceConfig) {
  */
 MongoStore.prototype.populate = function(callback) {
   var self = this;
+  if (!self._db) return;
```

Comment:
Additionally `console.error` something here instead of just failing silently so the user isn't left guessing?